packages/tracker/tracker.js

```diff
@@ -453,9 +453,13 @@ Tracker.flush = function (_opts) {
 // so that it is stopped if the current computation is invalidated.
 
 /**
+ * @callback TrackerComputationFunction
```

Comment:
Do you think it would be better to namespace it? Like Tracker.ComputationFunction?